### ISSUE TITLE
Can the colors of the table headers in Relationships view be configured?

### ISSUE THREAD

**Issue by bobhairgrove** [schemaspy/schemaspy#981]
I really liked the older "green" style of the ER relationships, [for example, like this...](https://dbmstools.com/storage/screenshots/schemaspy-tzvqdy52bhri54ap.jpeg)

But it looks like everything is already computed by GraphViz and output as a PNG file, so no possibility of doing this with a custom CSS stylesheet.

Can this be changed without hacking the sources?

**Comment by npetzall** [comment]
Yes, you can change table and table header using a custom css when building.

I'll add instruction when I get to a computer.

**Comment by npetzall** [comment]
So the simplest thing I can think of is to download and add 
 https://github.com/schemaspy/schemaspy/blob/master/src/main/resources/layout/schemaSpy.css in the same directory as Schemaspy-jar

Then you can modify 
https://github.com/schemaspy/schemaspy/blob/5cfa59137b868fb6e3bfd3a5deffabcb30359139/src/main/resources/layout/schemaSpy.css#L9-L13

To change the header of a table.

https://github.com/schemaspy/schemaspy/blob/5cfa59137b868fb6e3bfd3a5deffabcb30359139/src/main/resources/layout/schemaSpy.css#L26-L28

To change background for indexed column (I assume this will include PK and FK). 

There is a primaryKey class but I can't see that it's used.
